### PR TITLE
refactor(get_all_samples function names)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

rename get_all_samples_xxx to get_samples_xxx

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [x] Ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] Paxa the environment: `paxa`
- [x] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b refactor-sample-function-names -a
    ```

### How to test

- [x] Do cg get sample
- [x] do cg transfer 
- [x] do cg workflow balsamic start modestjaguar

### Expected test outcome

- [x] Check that ...
- [x] Take a screenshot and attach or copy/paste the output.

## Review

- [x] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: tests/meta/orders/test_meta_orders_api.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def test_submit_illegal_sample_customer(
         invoice_reference="dummy nr",
     )
     sample_store.add_commit(new_customer)
-    existing_sample = sample_store.get_all_samples()[0]
+    existing_sample = sample_store.get_samples()[0]
```
Comment (henrikstranneheim) [inline tests/meta/orders/test_meta_orders_api.py:158]: Add type hint

File: tests/meta/orders/test_meta_orders_status.py [hunk 2]
```diff
@@ -199,7 +199,7 @@ def test_store_rml(orders_api, base_store, rml_status_data, ticket: str):
     # GIVEN a basic store with no samples and a rml order
     assert base_store._get_query(table=Pool).count() == 0
     assert base_store.families().count() == 0
-    assert len(base_store.get_all_samples()) == 0
+    assert len(base_store.get_samples()) == 0
```
Comment (henrikstranneheim) [inline tests/meta/orders/test_meta_orders_status.py:202]:
```suggestion
    assert not base_store.get_samples()
```

File: tests/meta/orders/test_meta_orders_status.py [hunk 3]
```diff
@@ -248,7 +248,7 @@ def test_store_rml(orders_api, base_store, rml_status_data, ticket: str):
 
 def test_store_samples(orders_api, base_store, fastq_status_data, ticket: str):
     # GIVEN a basic store with no samples and a fastq order
-    assert len(base_store.get_all_samples()) == 0
+    assert len(base_store.get_samples()) == 0
```
Comment (henrikstranneheim) [inline tests/meta/orders/test_meta_orders_status.py:251]:
```suggestion
    assert not base_store.get_samples()
```

File: tests/meta/orders/test_meta_orders_status.py [hunk 4]
```diff
@@ -278,7 +278,7 @@ def test_store_samples(orders_api, base_store, fastq_status_data, ticket: str):
 
 def test_store_samples_sex_stored(orders_api, base_store, fastq_status_data, ticket: str):
     # GIVEN a basic store with no samples and a fastq order
-    assert len(base_store.get_all_samples()) == 0
+    assert len(base_store.get_samples()) == 0
```
Comment (henrikstranneheim) [inline tests/meta/orders/test_meta_orders_status.py:281]:
```suggestion
    assert base_store.get_samples()
```

File: tests/meta/orders/test_meta_orders_status.py [hunk 5]
```diff
@@ -298,7 +298,7 @@ def test_store_samples_sex_stored(orders_api, base_store, fastq_status_data, tic
 
 def test_store_fastq_samples_non_tumour_wgs_to_mip(orders_api, base_store, fastq_status_data):
     # GIVEN a basic store with no samples and a non-tumour fastq order as wgs
-    assert len(base_store.get_all_samples()) == 0
+    assert len(base_store.get_samples()) == 0
```
Comment (henrikstranneheim) [inline tests/meta/orders/test_meta_orders_status.py:301]: Replace this to use `not` instead for all occurrences.

**Review by henrikstranneheim — APPROVED** [review]
Great! 👍

**Comment by ChrOertlin** [comment]
`cg get sample lovelyguidedosprey`
+--------------------+---------------------+------------+---------------+--------------------+------------+-------------+
| Sample             | Name                | Customer   | Application   | State              | Priority   | External?   |
|--------------------+---------------------+------------+---------------+--------------------+------------+-------------|
| lovelyguidedosprey | testiesttest_sample | cust000    | WGSPCFC030    | Ordered 2023-03-22 | standard   | No          |
+--------------------+---------------------+------------+---------------+--------------------+------------+-------------+

`cg transfer lims --sample-id lovelyguidedosprey`
1 samples to process

`cg workflow balsamic start modestjaguar`
Case modestjaguar exists in status db
Fetch latest version from bundle oddlyprimeiguana
Fetch latest version from bundle oddlyprimeiguana
Linking fastq files in housekeeper for case modestjaguar
Fetch latest version from bundle oddlyprimeiguana
Found file oddlyprimeiguana/2021-10-19/H5M5NDSX2_ACC7948A21_S456_L001_R1_001.fastq.gz
Found file oddlyprimeiguana/2021-10-19/H5M5NDSX2_ACC7948A21_S456_L001_R2_001.fastq.gz
Fetch latest version from bundle oddlyprimeiguana
Decompression for case modestjaguar not needed
Case modestjaguar exists in status db
Fetching files from bundle oddlyprimeiguana
Fetching files with tags in [fastq]
Linking: /home/proj/stage/housekeeper-bundles/oddlyprimeiguana/2021-10-19/H5M5NDSX2_ACC7948A21_S456_L001_R1_001.fastq.gz -> /home/proj/stage/cancer/cases/modestjaguar/fastq/1_171015_H5M5NDSX2_oddlyprimeiguana_XXXXXX_R_1.fastq.gz
Linking: /home/proj/stage/housekeeper-bundles/oddlyprimeiguana/2021-10-19/H5M5NDSX2_ACC7948A21_S456_L001_R2_001.fastq.gz -> /home/proj/stage/cancer/cases/modestjaguar/fastq/1_171015_H5M5NDSX2_oddlyprimeiguana_XXXXXX_R_2.fastq.gz
Concatenation in progress for sample oddlyprimeiguana.
Concatenating: 1_171015_H5M5NDSX2_oddlyprimeiguana_XXXXXX_R_1.fastq.gz -> concatenated_oddlyprimeiguana_XXXXXX_R_1.fastq.gz
Concatenation file size check successful!
Concatenating: 1_171015_H5M5NDSX2_oddlyprimeiguana_XXXXXX_R_2.fastq.gz -> concatenated_oddlyprimeiguana_XXXXXX_R_2.fastq.gz
Concatenation file size check successful!
Creating config file for modestjaguar.
Case modestjaguar exists in status db
Fetching files from bundle oddlyprimeiguana
Fetching files with tags in [fastq]
Case modestjaguar has following BALSAMIC samples:
SAMPLE ID            TISSUE TYPE          APPLICATION          BED VERSION         
oddlyprimeiguana     tumor                tgs                  gmsmyeloid_5.2_hg19_design.bed

The provided gender is unknown, setting female as the default
Running command /home/proj/stage/bin/miniconda3/envs/S_balsamic/bin/balsamic config case --analysis-dir /home/proj/stage/cancer/cases --balsamic-cache /home/proj/stage/cancer/balsamic_cache/ --case-id modestjaguar --gender female --analysis-workflow balsamic --genome-version hg19 --tumor /home/proj/stage/cancer/cases/modestjaguar/fastq/concatenated_oddlyprimeiguana_XXXXXX_R_1.fastq.gz --panel-bed /home/proj/stage/cancer/reference/panel/gmsmyeloid_5.2_hg19_design.bed --tumor-sample-name oddlyprimeiguana --swegen-snv /home/proj/stage/cancer/reference/swegen/grch37_swegen_1k_snv_-20170830-.vcf.gz --swegen-sv /home/proj/stage/cancer/reference/swegen/grch37_swegen_1k_sv_-20170830-.vcf.gz --clinical-snv-observations /home/proj/stage/cancer/reference/loqusdb/loqusdb_clinical_snv_variants_export-20221010-.vcf.gz --clinical-sv-observations /home/proj/stage/cancer/reference/loqusdb/loqusdb_clinical_sv_variants_export-20221010-.vcf.gz

**Pull request merged**